lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -327,6 +328,8 @@ def main():
                     dl1_container['disp_angle'] = disp_angle
                     dl1_container['disp_sign'] = disp_sign
 
+                dl1_container['sin_az_tel'] = np.sin(infile.root[dl1_params_lstcam_key].col('az_tel'))
+                    
```

Comment:
Hum, this is setting a whole column to a single event value (where there should be a float)

Reply:
I think you want
```
dl1_container['sin_az_tel'] = np.sin(infile.root[dl1_params_lstcam_key].col('az_tel')[ii])
```

Reply:
or even
```
dl1_container['sin_az_tel'] = np.sin(params['az_tel'][ii])
```